test/parse_test.js

```diff
@@ -103,19 +103,44 @@ describe('parse()', function() {
       title: 'There were errors validating the AsyncAPI document.',
       parsedJSON: JSON.parse(invalidYamlOutput),
       validationErrors: [
+        {
+          title: '/ should NOT have additional properties',
+          location: {
+            jsonPointer: '/test',
+            startLine: 15,
+            startColumn: 1,
+            startOffset: offset(305,15),
+            endLine: 15,
+            endColumn: 11,
+            endOffset: offset(315,15)
+          }
+        },
+        {
+          title: '/info should NOT have additional properties',
```

Comment:
just to make sure. To get information here, in the info object, about the additional property we would need to start reading `params: { additionalProperty: 'test' }` from ajv?

Reply:
no wait, sorry, you already read it here https://github.com/asyncapi/parser-js/pull/280/files#diff-5dfe38baf287dcf756a17c2dd63483781b53bf4b669e10efdd01e74bcd8e780aR245

but it is not used in this error because `error.dataPath` is probably there as it is a nested object. Do I get it right?

Reply:
@derberg If the additional property is not in the root, then `error.dataPath` contains the path to its parent. But if it is in the root, it has no parent and `error.dataPath` will be an empty string. I solved the problem by passing the additional property to `findLocationOf` (instead of `error.dataPath`). then `findLocationOf` locates the additional property and returns its location.

Reply:
then in my opinion we should concat `error.dataPath` with `additionalProperty` if both are available, so later you get a helpful error that point to `/info/test`. Makes sense?

Reply:
Yes. pushed the changes. works perfectly.